### PR TITLE
ADEN-5601 Disable Veles on pages with featured video

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

**Comment by dianafa** [comment]
LGTM

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusAst.js [hunk 1]
```diff
@@ -26,7 +27,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusAst',[
 		};
 
 	function isEnabled() {
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusAst.js]: It looks like context variable

Comment (v3rth) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusAst.js:27]: I guess this function is used somewhere inside prebid, but anyway I will consider to extract it to `adContext.js` or all of prebid related flags to some "special" biddersAdContext

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js [hunk 2]
```diff
@@ -32,7 +33,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.rubicon', [
 		};
 
 	function isEnabled() {
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/rubicon.js:33]: Same here

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js [hunk 3]
```diff
@@ -42,8 +42,10 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.veles', [
 		};
 
 	function isEnabled() {
```
Comment (v3rth) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js:42]: ...aaaanndd here :)

**Review by v3rth — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/0293e97ce5ea4cd27e72ea4d63ff0a6a585dc06c/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1064 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by v3rth — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/0ce96ea6c5982021d4e3d5435d1406b17926afab/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1100 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdContext.spec.js](https://github.com/Wikia/app/blob/948555d2efcffe45e2118d002cf057e7f2fd2080/extensions/wikia/AdEngine/js/spec/AdContext.spec.js): File "AdContext.spec.js" has 1100 lines, which is greater than 1000 authorized. Split it into smaller files. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS104)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**